images/azure/rocky-91.yaml

```diff
@@ -0,0 +1,13 @@
+---
+download_images: true
+
+packer:
+  distribution: "rockylinux-9"                # Offer
+  distribution_version: "rockylinux-9"  # SKU
+  image_publisher: "erockyenterprisesoftwarefoundationinc1653071250513"
```

Comment:
This is ID for the official free rocky linux image template. https://portal.azure.com/#view/Microsoft_Azure_Marketplace/GalleryItemDetailsBladeNopdl/id/erockyenterprisesoftwarefoundationinc1653071250513.rockylinux-9

Reply:
🤮